doc/specs/Gif-Maker.md

```diff
@@ -0,0 +1,108 @@
+# GIF Maker Spec
+
+![Terminate](../images/Logo.jpg "Power Toys")
+
+- **What is it:** Users can record their screen and turn the recording into a GIF.
+- **Authors:** Benjamin Leverette and Prudence Phillips
+- **Spec Status:** Draft
+
+## 1. Overview
+
+### 1.1. Elevator Pitch / Narrative
+
+ David loves using GIFs to add creativity ti his typical responses to messages.  His friend Amanda emails him and asks what he would like to eat for lunch today.  David recently watched a burrito video, so he uses the GIF maker to screen record a brief moment from the video in order to tell Amanda that its "Burrito Day."
```

Comment:
`creativity ti his` -> `creativity to his`
`its "Burrito Day` -> `it's "Burrito Day`